sympy/core/numbers.py

```diff
@@ -19,6 +19,7 @@
 
 import mpmath
 import mpmath.libmp as mlib
+import numbers
```

Comment:
```suggestion
import numbers as num
```
You might want to import it as num since you are working in a module of the same name